InsuranceAgent/en/training.json

```
@@ -71,25 +71,12 @@
                     "expressions": [
                         "Help",
                         "Help me, please.",
-                        "I need some help",
-                        "What do you do",
-                        "What can you do",
-                        "how does this work",
-                        "what should I do",
-                        "who are you",
-                        "What's your purpose",
-                        "I don't know what to do",
-                        "what is this",
+                        "I need some help !",
```

This should probably be changed across all EN trainings
```suggestion
                        "I need some help!",
```